3d_segmentation/ignite/unet_evaluation_array.py

```diff
@@ -66,17 +66,20 @@ def main(tempdir):
     roi_size = (96, 96, 96)
     sw_batch_size = 4
 
+    post_trans = Compose([Activations(sigmoid=True), AsDiscrete(threshold_values=True)])
+
     def _sliding_window_processor(engine, batch):
         net.eval()
         with torch.no_grad():
             val_images, val_labels = batch[0].to(device), batch[1].to(device)
             seg_probs = sliding_window_inference(val_images, roi_size, sw_batch_size, net)
+            seg_probs = post_trans(seg_probs)
```

Comment:
As the output is already through `post_transform`, no need to call `predict_segmentation()` in SegmentationSaver anymore, please update all the evaluation examples if didn't delete it.
Thanks.